app/javascript/app/data/initial-state.js

```diff
@@ -1,10 +1,238 @@
 export default {
+  nav: {
+    countriesOpen: false
+  },
+  map: {
+    center: [0, 20],
+    zoom: 1
+  },
   countryExplorer: {
     open: false
   },
   countryNDC: {
     loading: false,
     loaded: false,
     data: {}
+  },
+  countrySelect: {
+    query: '',
+    countries: [
```

Comment:
One comment regarding big `initialState` files, we can chop the parts into individual files and export all of them merged as one data structure if it starts growing a lot.